src/take.js

```diff
@@ -20,13 +23,21 @@ var slice = require('./slice');
  * @return {Array}
  * @example
  *
- *      R.take(3,[1,2,3,4,5]); //=> [1,2,3]
+ *      R.take(1, ['foo', 'bar', 'baz']); //=> ['foo']
+ *      R.take(2, ['foo', 'bar', 'baz']); //=> ['foo', 'bar']
+ *      R.take(3, ['foo', 'bar', 'baz']); //=> ['foo', 'bar', 'baz']
+ *      R.take(4, ['foo', 'bar', 'baz']); //=> ['foo', 'bar', 'baz']
+ *      R.take(3, 'ramda');               //=> 'ram'
  *
- *      var members= [ "Paul Desmond","Bob Bates","Joe Dodge","Ron Crotty","Lloyd Davis","Joe Morello","Norman Bates",
- *                     "Eugene Wright","Gerry Mulligan","Jack Six","Alan Dawson","Darius Brubeck","Chris Brubeck",
- *                     "Dan Brubeck","Bobby Militello","Michael Moore","Randy Jones"];
+ *      var members = [
+ *        'Paul Desmond', 'Bob Bates', 'Joe Dodge', 'Ron Crotty',
+ *        'Lloyd Davis', 'Joe Morello', 'Norman Bates', 'Eugene Wright',
+ *        'Gerry Mulligan', 'Jack Six', 'Alan Dawson', 'Darius Brubeck',
+ *        'Chris Brubeck', 'Dan Brubeck', 'Bobby Militello', 'Michael Moore',
+ *        'Randy Jones'
+ *      ];
```

Comment:
I don't know these people, so I don't know whether the entire list is significant. If not, could we have the list end at Darius Brubeck for the sake of brevity?